### PR TITLE
doca2.6 support and work in parallel

### PR DESCRIPTION
**Pull request by masaruito110**
The environment is below

```
ubuntu 22.04
kernel 6.5.0-18-generic

$ nvidia-smi
Thu Feb 22 17:40:00 2024
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 545.23.08              Driver Version: 545.23.08    CUDA Version: 12.3     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA A100-PCIE-40GB          On  | 00000000:81:00.0 Off |                    0 |
| N/A   23C    P0              34W / 250W |      4MiB / 40960MiB |      0%      Default |
|                                         |                      |             Disabled |
+-----------------------------------------+----------------------+----------------------+

$ dpkg -l | grep doca
ii  doca-apps                             2.6.0058-1                               amd64        DOCA-Based reference applications
ii  doca-apps-dev                         2.6.0058-1                               amd64        Development files for DOCA Apps
ii  doca-extra                            0.1.4-1                                  amd64        A list of extra support scripts
ii  doca-gpu                              2.6.0058-1                               amd64        GPU runtime capabilities for DOCA
ii  doca-gpu-dev                          2.6.0058-1                               amd64        Development files for DOCA GPU
ii  doca-grpc                             2.6.0058-1                               amd64        gRPC runtime capabilities for DOCA
ii  doca-grpc-dev                         2.6.0058-1                               amd64        Development files for DOCA grpc
ii  doca-host-repo-ubuntu2204             2.6.0-0.0.1.2.6.0058.1.24.01.0.3.3.1     amd64        Doca repo bundle package
ii  doca-libs                             2.6.0058-1                               amd64        Data Center on a Chip Architecture (DOCA)
ii  doca-prime-runtime                    2.6.0058-1                               amd64        DOCA prime runtime metapackage
ii  doca-prime-sdk                        2.6.0058-1                               amd64        DOCA prime sdk metapackage
ii  doca-prime-tools                      2.6.0058-1                               amd64        Runtime DOCA Tools
ii  doca-samples                          2.6.0058-1                               amd64        DOCA Samples
ii  doca-sdk                              2.6.0-0.0.1                              amd64        doca-sdk meta-package
ii  doca-services                         2.6.0058-1                               amd64        Runtime utilities associated with DOCA Services
ii  libdoca-libs-dev                      2.6.0058-1                               amd64        Development files for DOCA Libs
```

### CODE REVIEW

File: src/doca-util.cc [hunk 1]
```diff
@@ -1361,13 +1378,13 @@ create_root_pipe(struct rxq_tcp_queues* tcp_queues, struct doca_flow_port* port)
...
 
-std::string doca_error_get_descr(doca_error_t err)
-{
-#ifdef DOCA22
-    return doca_get_error_string(err);
-#else
-    return doca_error_get_descr(err);
-#endif
-}
+// std::string doca_error_get_descr(doca_error_t err)
```
[the remark this answers is not in the document]
Comment (masaruito110) [reply src/doca-util.cc:1381]: It causes compile errors. So I already modified test code to pass compile

[the remark this answers is not in the document]
Comment (Fixstars-iizuka) [reply src/doca-util.cc:1381]: Ah understood. Then can we remove DOCA22/25 related code entirely?

[the remark this answers is not in the document]
Comment (masaruito110) [reply src/doca-util.cc]: removed!

**Comment by iitaku** [comment]
LGTM

**Pull request merged**